lib/MySQL_Variables.cpp

```diff
@@ -0,0 +1,358 @@
+#include "MySQL_Variables.h"
+#include "proxysql.h"
+
+#include "MySQL_Session.h"
+#include "MySQL_Data_Stream.h"
+#include "SpookyV2.h"
+
+#include <sstream>
+
+MySQL_Variables::MySQL_Variables(MySQL_Session* _session) {
+	assert(_session);
+	session = _session;
```

Comment:
I have noticed that members variables are never referred as "this" is this a convention? It's sometimes handy for me while browsing code, I can easily see which are the "member variables" vs "local ones".

Reply:
if this == NULL then you will have a crash